presto-main/src/test/java/com/facebook/presto/operator/scalar/TestScalarValidation.java

```diff
@@ -110,7 +110,7 @@ public static long bad()
         }
     }
 
-    @Test(expectedExceptions = IllegalArgumentException.class, expectedExceptionsMessageRegExp = "Method .* has parameter with wrapper type Boolean that is missing @SqlNullable")
+    @Test(expectedExceptions = IllegalStateException.class, expectedExceptionsMessageRegExp = "Method .* has parameter expected to use USE_NULL_FLAG or RETURN_NULL_ON_NULL null convention. Parameter type is unexpected: Boolean")
```

Comment:
The assertions should have use

```
checkCondition(..., FUNCTION_IMPLEMENTATION_ERROR, ...)
```

Fix all in `ScalarFromAnnotationsParser` and `ParametricScalarImplementation` after this PR is merged.